week16/형민/PG81303_표편집.java

```diff
@@ -0,0 +1,50 @@
+import java.io.*;
+import java.util.*;
+
+class Solution {
+
+    static public String solution(int n, int k, String[] cmd) {
+        LinkedList<Character> linkedList = new LinkedList<>();
+        for (int i = 0; i < n; i++) {
+            linkedList.addLast('O');
+        }
+
+        Stack<Integer> stack = new Stack<>();
+        int moveCnt = 0;
+        for(String cm : cmd) {
+            char c = cm.charAt(0);
+
+            if(c == 'U') {
+                moveCnt = cm.charAt(2) - '0';
+                k -= moveCnt;
+            } else if(c == 'D') {
+                moveCnt = cm.charAt(2) - '0';
+                k += moveCnt;
+            } else if(c == 'C') {
+                if(k == linkedList.size() - 1) {
+                    linkedList.remove(k);
+                    stack.push(k);
+                    k -= 1;
+                } else {
+                    linkedList.remove(k);
+                    stack.push(k);
+                }
```

Comment:
약간 중복적인 내용을 축약해도 좋을 것 같아요!
```suggestion
                linkedList.remove(k);
                stack.push(k);
                if(k == linkedList.size() - 1) {
                    k -= 1;
                }
```